README.md

```diff
@@ -65,8 +72,6 @@ You have to ensure that your local environment has the following:
 - Ensure the `JAVA_HOME` environment variable your terminal is configured to point to JDK17.
 - Compile the project using `build/sbt package`
```

Comment:
Is this still required?

Reply:
Is `build/sbt package` required to have a dockerized environment? I don't think so as `sbt package` is part of building the UC images.

Reply:
Having the `build/sbt` package would be helpful for anyone building apps though.  Not a blocker though.